app-android/src/main/res/values/themes.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 
-    <style name="Theme.KaigiApp" parent="android:Theme.Material.Light.NoActionBar">
+    <style name="Theme.KaigiApp" parent="Theme.AppCompat.Light.NoActionBar">
```

Comment:
🙏

Reply:
let me check with revert ver 🙇

Reply:
I don't have to change the theme for entire the app.

I can just update the theme for the `Activities` which generate by the plugin.